### ISSUE TITLE
Ne pas avoir de doublon dans les personnes à notifier

### ISSUE THREAD

**Issue by yaf** [betagouv/rdv-solidarites.fr#2137]
## Actuellement

Pas tout à fait un bug... ou peut-être un bug ?

Dans un cas de figure un peu particulier, nous pouvons avoir deux fois les mécanismes de notification pour la même personne (cas d'un proche et de sont responsable dans le même RDV).

![Screenshot 2022-02-09 at 17-21-33 Votre agendaRDV Patricia DUROY, Joséphine DUROY, Paul Emile DASILVA DACOSTA et Léa DUPONT](https://user-images.githubusercontent.com/42057/153244613-802de095-fc8a-4956-a011-7d7442addd69.png)

## Objectif

Finalement, ça ne devrait pas avoir un impact fort sur les notifications, mais il serait peut-être intéressant d'envisager la gestion des doublons à l'affichage ? (Pour les notifications, c'est à vérifier, mais je crois que c'est déjà réglé :thinking: )

**Comment by n-b** [comment]
Yes, je m’étais fait la réflexion. En fait, je pense qu’il faudrait faire une passe de redesign sur la vue de détail du Rdv. C’est ce genre de constat qui me pousse à renommer `RdvUser` en “participation”.
